Shared/Controls/ImageSlideshow/ImageSlideshow.cs

```diff
@@ -301,7 +315,14 @@ private void CheckAndSetSelectedIndexOnFlipView()
 				}
 				else
 				{
-					Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, async () =>
+#if WINDOWS10_0_18362_0
```

Comment:
... but here it's WINDOWS10_0_18362_0?

Reply:
Yet the content is in the condition is the same.

Reply:
This was a mistake, the usage of DispatcherQueue is only really necessary for the Windows App, the correct condition was for WINDOWS10_0_18362_0